db_stress_tool/db_stress_common.h

```diff
@@ -197,11 +197,10 @@ DECLARE_int32(prefix_size);
 DECLARE_bool(use_merge);
 DECLARE_bool(use_full_merge_v1);
 
-static const long KB = 1024;
-static const int kRandomValueMaxFactor = 3;
-static const int kValueMaxLen = 100;
+const long KB = 1024;
```

Comment:
Do we need `extern` here?

Reply:
I think we don't need it for constants. Maybe I'm wrong. No compiler complains it though.